docs/manual/cluster-admin/troubleshooting.md

```diff
@@ -23,7 +23,7 @@ Solutions:
 This is a kind of alert from alert manager, and is reported by watchdog service. Watchdog gets such metrics from Kubernetes API. Example metrics is like:
 
 ```
-pai_node_count{disk_pressure="false",instance="10.0.0.1:9101",job="pai_serivce_exporter",memory_pressure="false",name="10.0.0.2",out_of_disk="false",pai_service_name="watchdog",ready="true",scraped_from="watchdog-5ddd945975-kwhpr"}
+pai_node_count{disk_pressure="false",instance="10.0.0.1:9101",job="pai_serivce_exporter",memory_pressure="false",host_ip="10.0.0.2",out_of_disk="false",pai_service_name="watchdog",ready="true",scraped_from="watchdog-5ddd945975-kwhpr"}
```

Comment:
Can we change to nodeName. Which is more easy to understand

Reply:
> Can we change to `node_name`?. Which is more easy to understand

Added `node_name` as another label. Used `node_name` instead in inhibit rules.

Reply:
Add `node_name` in this doc?